tests/global_config.py

```diff
@@ -166,6 +166,9 @@ def _get_default_storage_class(sc_list):
 default_volume_mode = default_storage_class_configuration["volume_mode"]
 default_access_mode = default_storage_class_configuration["access_mode"]
 
+storage_class_for_storage_migration_a = HppCsiStorageClass.Name.HOSTPATH_CSI_BASIC
```

Comment:
What about `storage_class_for_storage_migration_source` and `storage_class_for_storage_migration_destination`?

Reply:
Different clusters may have different storage classes; HPP and OCS are just defaults, but they may be overridden via command line according to the environment where the tests are running.